test/hdimage6.fdisk

```diff
@@ -0,0 +1,6 @@
+images/test.hdimage1:start=256,size=512,type=83
+images/test.hdimage2:start=768,size=6144,type=83
+images/test.hdimage3:start=7199,size=79363,type=f
+images/test.hdimage5:start=7200,size=15872,type=83
+images/test.hdimage6:start=23088,size=31744,type=83
+images/test.hdimage7:start=54848,size=31744,type=83
```

Comment:
I'm quite certain, that start + size for partition 3 (the extended partition) should be the same as start + size for partition 7 (the last logical partition). But that bug is not new.

Reply:
That sound logical, indeed